index.html

```diff
@@ -408,6 +408,28 @@ <h1>Introduction</h1>
+      <ul>
+        <li><strong>Manufactured.</strong> At this stage the WoT device is manufactured by an OEM with a variant of WoT stack enabled. It might have a WoT runtime present that enables single or multi-tenant users or it might only have exposed WoT interfaces present. From manufactured state, the device can transition to either of two states: Installation/Commissioning state or Security Provisioning state based on deployment scenario.</li>
+        <li><strong>Installation/Commissioning</strong> At this stage a WoT device is installed into its intended deployment scenario by a system provider or system integrator, as well as necessary SW configuration is performed. This in practice can be combined with the following Security Provisioning stage.</li>
+        <li><strong>Security Provisioning.</strong> The main goal of this stage is to provision a WoT device with a required set of security metadata and credentials that it needs during the Operational state. The exact set of these credentials would vary based on deployment model, choice of security mechanisms etc. <cite> The State-of-the-Art and Challenges for the Internet of Things Security</cite> [[Garcia17]] document refers to this stage as "Bootstrapping".</li>
+        <li><strong>Operational State.</strong> This is the default operational state of WoT devices. Devices are fully ready to communicate over WoT interfaces and operate to their deployment scenarios. From Operational state a device can return back to manufactured state if it is fully decommissioned and security de-provisioned (link E in Figure <a href="#wot-lifecycle"></a>). It is also possible that only partial reset on device is performed: if it only requires re-installation and recommissioning without security re-provisioning needed (link D in Figure <a href="#wot-lifecycle"></a>) or only security re-provisioning is needed without re-installation and re-commissioning (link C in Figure <a href="#wot-lifecycle"></a>).</li>
+        <li><strong>Maintenance and security updates.</strong> This is an optional state that can be activated remotely or locally (link A in Figure <a href="#wot-lifecycle"></a>) on a WoT device depending on the deployment model and overall setup. This state can be required for example to update the SW layers of a WoT device or perform some security-related updates, such as key revocation, certificate updates etc. After the maintenance or updates are done, the device returns back to the Operational state (link B in Figure <a href="#wot-lifecycle"></a>). </li>
+      </ul>
+      The scope of this document only covers the Operation state of a WoT device with the assumption that all relevant security metadata and credentials has been successfully provisioned during the Security Provisioning state and available transparently to the WoT runtime level. The security-related updates (such as revocation and update of keys and other credentials) is also out of the scope of this document.
+      </p>

```

Comment:
Perhaps it would be nice listing some  examples from various IoT standards. I guess there are not so many options. Perhaps links would be enough.